### PR TITLE
Publish kubed chart

### PR DESCRIPTION
**Pull request by tamalsaha**
https://github.com/appscode/kubed
```
Kubed (pronounced Cube-Dee) by AppsCode is a Kubernetes Cluster Operator Daemon. Kubed can do the following things for you:

Kubed can take periodic snapshot of a Kubernetes cluster objects.
Kubed provides a recycle bin for deleted and/or updated Kubernetes objects.
Kubed can keep ConfigMaps and Secrets synchronized across Namespaces.
Kubed can forward cluster events to various destinations.
Kubed can setup janitors for Elasticsearch and InfluxDB.
Kubed can send notifications via Email, SMS or Chat.
Kubed has a built-in search engine for your cluster objects using bleve.
```

### CODE REVIEW

File: stable/kubed/README.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+$ helm install stable/kubed
+```
+
+## Introduction
+
+This chart bootstraps a [Kubed controller](https://github.com/appscode/kubed) deployment on a [Kubernetes](http://kubernetes.io) cluster using the [Helm](https://helm.sh) package manager.
+
+## Prerequisites
+
+- Kubernetes 1.5+
```
Comment (unguiculus) [inline stable/kubed/README.md]: 1.7+ is officially supported.

File: stable/kubed/templates/deployment.yaml [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+      serviceAccountName: {{ if .Values.rbac.create }}{{ template "kubed.fullname" . }}{{ else }}"{{ .Values.rbac.serviceAccountName }}"{{ end }}
+      containers:
+      - name: kubed
+        args:
+        - run
+        - --v={{ .Values.logLevel }}
+        - --api.address=:8080
+        - --web.address=:56790
+        image: '{{ .Values.kubed.image }}:{{ .Values.kubed.tag }}'
+        imagePullPolicy: {{ default "" .Values.imagePullPolicy | quote }}
```
Comment (unguiculus) [inline stable/kubed/templates/deployment.yaml]: Remove `default` function. It's unnecessary.

**Review by unguiculus — CHANGES_REQUESTED** [review]

**Comment by tamalsaha** [comment]
Addressed comments. PTAL.

**Review by unguiculus — APPROVED** [review]

**Pull request merged**